.github/CODEOWNERS

```diff
@@ -1318,6 +1318,12 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 #CC# /src/plugins/kibana_react/public/code_editor/ @elastic/kibana-presentation
 
 # Machine Learning
+/x-pack/test/functional_with_es_ssl/apps/discover_ml_uptime/config.ts @elastic/ml-ui
+/x-pack/test/stack_functional_integration/apps/ml @elastic/ml-ui
+/x-pack/test/monitoring_api_integration/fixtures/elasticsearch/ml_jobs.json @elastic/ml-ui
+/x-pack/test/monitoring_api_integration/apis/elasticsearch/ml_jobs.ts @elastic/ml-ui
```

Comment:
As discussed in slack, I think the whole directory `/x-pack/test/monitoring_api_integration` should be owned by `@elastic/stack-monitoring`

Reply:
Whoops, I'd built these entries previously.  Will fixup

Reply:
Already [added](https://github.com/elastic/kibana/pull/200217) to the stack monitoring pr

Reply:
[9c63452](https://github.com/elastic/kibana/pull/200212/commits/9c634521edb5cb1de8495bc0472124e3d8965777)